### ISSUE TITLE
hot and inline options don't work as expected. 

### ISSUE THREAD

**Issue by octalmage** [webpack-contrib/grunt-webpack#121]
When setting both inline and hot to true in the webpack-dev-server task, HMR throws this error: 

```
[HMR] Hot Module Replacement is disabled.
```

I believe this is due to this line here: https://github.com/webpack-contrib/grunt-webpack/blob/master/tasks/webpack-dev-server.js#L67

When I comment out that line, and add the HMR plugin on my own, it works fine but with this error: 

```
app.js?ver=4.7.2:891 Uncaught RangeError: Maximum call stack size exceeded
``` 

Which appears to be caused by multiple versions of the HMR plugin (makes sense). 

If I only enable inline and set hot to false (without manually loading the plugin), it reloads the page fine. It's only when both inline and hot are set to true that it breaks.

### Webpack/Grunt Configuration (webpack.config.js, webpack part of Gruntfile.js)

Grunt task: 
```JavaScript
  'webpack-dev-server': {
    options: {
      webpack: webpackConfig,
      publicPath: 'http://localhost:8888/inc/js/',
      inline: true,
      port: 8888,
      hot: true,
      host: '0.0.0.0',
      watchOptions: {
        poll: true,
        ignored: /node_modules/
      }
    },
    start: {
     webpack: {
       devtool: 'eval',
       plugins: webpackConfig.plugins.concat( // Add the HotModule plugin.
         new webpack.HotModuleReplacementPlugin()
       )
     },
   }
  },
```

Webpack Config: 
```JavaScript
var path = require('path');
var webpack = require('webpack');
var ExtractTextPlugin = require('extract-text-webpack-plugin');

module.exports = {
  entry: './js/app/application.js',
  output: {
    path: path.join(__dirname, 'inc/js'),
    filename: 'app.js',
    publicPath: 'http://localhost:8888/inc/js/',
  },
  module: {
    rules: [
      { test: /\.js$/, use: ['babel-loader', 'react-hot-loader'], exclude: /node_modules/ },
      {
        test: /\.scss$/,
        loader: ExtractTextPlugin.extract({
          fallback: 'style-loader',
          use: 'css-loader!sass-loader',
        }),
      },
    ]
  },
  devtool: 'source-map',
  plugins: [
    new ExtractTextPlugin('app.css')
  ]
}
```

### Expected Behavior
I expect HMR to work without needing to load the HMR plugin multiple times. 

### Current Behavior
Instead I have to manually load the HMR plugin, which is fine, but I wish the Grunt task didn't. 

### Possible Solution
Remove [this](https://github.com/webpack-contrib/grunt-webpack/blob/master/tasks/webpack-dev-server.js#L67), that seems to work for me. I don't mind manually adding the plugin to the config.  

### Your Environment

| software         | version
| ---------------- | -------
| webpack          | 2.2.1
| grunt-webpack    | 2.0.1
| grunt            | 0.4.5
| node             | 7.5.0
| npm              | 4.1.2
| Operating System | Ubuntu Docker: https://hub.docker.com/r/_/wordpress/

Thanks! It's possible that this is caused by Webpack 2, but I'm not sure.

**Comment by danez** [comment]
This is fixed with #126 and going to release in a minute.

**Comment by joaovieira** [comment]
@danez thanks 👍
